packages/babel-plugin-proposal-optional-chaining/package.json

```diff
@@ -21,6 +21,7 @@
   "devDependencies": {
     "@babel/core": "^7.9.0",
     "@babel/helper-plugin-test-runner": "^7.8.3",
-    "@babel/plugin-transform-block-scoping": "^7.8.3"
+    "@babel/plugin-transform-block-scoping": "^7.8.3",
+    "@babel/helper-get-call-context": "7.9.6"
```

Comment:
Use `^`. I don't think that it's what is causing problems, but it's the only difference I see.

Reply:
No difference unfortunately. Perhaps https://github.com/babel/babel/pull/11581 could be the fix?